src/kubernetes/server/server-deployment.yaml

```diff
@@ -64,7 +67,7 @@ spec:
         - name: SPRING_CLOUD_KUBERNETES_SECRETS_ENABLE_API
           value: 'false'
         - name: SPRING_CLOUD_KUBERNETES_SECRETS_PATHS
-          value: /etc/secrets
+          value: /etc/secrets,/workspace/runtime/secrets/ctr-registry
```

Comment:
probably not needed with the above change

Reply:
same, this was mistakenly committed and I removed it form the PR. The  server-deployment.yaml should be changed by this PR